src/components/text/index.tsx

```diff
@@ -1,14 +1,38 @@
 import * as React from 'react';
 import { styled } from 'linaria/react';
-import { sizes, colors } from 'styles';
+import { sizes } from 'styles';
 import { useTheme } from 'utils/hooks';
-export interface TextProps extends React.HTMLAttributes<HTMLSpanElement> {
+
+export interface TextProps extends React.HTMLAttributes<HTMLElement> {
   color?: any;
   weight?: any;
   size?: any;
+  component?: string;
 }
 
-const StyledSpan = styled.span`
+
+const StyledSpan =  styled.span`
+  display: inline-block;
+  font-size: ${(props: any): any => {
```

Comment:
this font size can also either be a string or a number, why any?

Reply:
copy paste